examples/api/test/rendering/sliver_grid/sliver_grid_delegate_with_fixed_cross_axis_count.0_test.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/rendering/sliver_grid/sliver_grid_delegate_with_fixed_cross_axis_count.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Each tiles should have a width of 200.0 and a height of 400.0', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      const example.SliverGridDelegateWithFixedCrossAxisCountExampleApp(),
+    );
+
+    for (int i = 0; i < 4; i++) {
```

Comment:
The example creates 20 items, maybe we should test all of them? As you want.

Reply:
There is no reason for not testing all of them so here you go [test: Test the 20 children](https://github.com/flutter/flutter/pull/153500/commits/7d94b83893b76d526b4aac37fec44a36c4142831) 😊